src/App.tsx

```diff
@@ -1,61 +1,41 @@
+import { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { PreparedTodo } from './types';
+
+import { TodoAddForm } from './components/TodoAddForm';
+import { TodoList } from './components/TodoList';
+
+const preparedTodoTasks: PreparedTodo[] = todosFromServer.map(todo => {
+  const user = usersFromServer.find(({ id }) => todo.userId === id) || null;
```

Comment:
Create separate function helper for this logic, I saw you have very similar logic below in your code